frontend/src/common/queries/common.ts

```diff
@@ -1,3 +1,3 @@
 export const DEFAULT_FETCH_OPTIONS = {
-  credentials: "include" as RequestCredentials,
+  credentials: "same-origin" as RequestCredentials,
```

Comment:
Is this going to prevent us from working on hosted dev from local dev?

Reply:
Thanks for the call out! I think given that [AuthN BE PR](https://github.com/chanzuckerberg/corpora-data-portal/pull/729) is still in flux and this PR will likely get merged first, temporarily having `same-origin` would allow FE to use Requestly to send cookies, instead of getting the CORS error from BE!